### ISSUE TITLE
Does not work with credentials set via environment variables

### ISSUE THREAD

**Issue by hffmnn** [teamplanes/cdk-watch#5]
Hi,

I am using `aws-vault` which sets up the shell environment variables for AWS when logging in. Running `yarn cdkw "**" ` after that gives 

```
...
code: 'CredentialsError',
  time: 2021-02-19T07:47:08.770Z,
  originalError: {
    message: 'Could not load credentials from SharedIniFileCredentials',
    code: 'CredentialsError',
    time: 2021-02-19T07:47:08.770Z,
    originalError: {
      message: 'Credentials not set for profile default',
      code: 'SharedIniFileCredentialsProviderFailure',
      time: 2021-02-19T07:47:08.769Z
    }
```
from somewhere in the aws sdk.
The docs of the sdk read like this. so I guess it should work (and e.g. the `cdk` command works that way)

```
If more than one credential source is available to the SDK, the default precedence of selection is as follows:

    Credentials that are explicitly set through the service-client constructor

    Environment variables

    The shared credentials file

    Credentials loaded from the ECS credentials provider (if applicable)

    Credentials that are obtained by using a credential process specified in the shared AWS config file or the shared credentials file. For more information, see Loading Credentials in Node.js using a Configured Credential Process.

    Credentials loaded from AWS IAM using the credentials provider of the Amazon EC2 instance (if configured in the instance metadata)
```

From https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html

Any ideas?

**Comment by hoegertn** [comment]
Should be solved with my PR #4 
The credentials chain is broken imho

**Comment by kirkness** [comment]
Correct, hoping to merge and release #4 today, which will fix this.

**Comment by hffmnn** [comment]
Cool. Very nice 👍

**Comment by kirkness** [comment]
Fixed in #4 👍
